### ISSUE TITLE
range doesnt handle case when n > m

### ISSUE THREAD

**Issue by kristiantoska** [andi7/utils#1]
![image](https://user-images.githubusercontent.com/20017227/57869882-3ffa0e80-7806-11e9-9b9a-dc13d583bf5e.png)

UNFORGIVABLE!!! CANT USE ANDI'S UTILS ANYMORE!

**Comment by ardibegolli** [comment]
Same thing happened to me! I guess I will go back to lodash again. Shame

**Comment by kristiantoska** [comment]
Yeah i really wanted to use this lib. I'll return to good 'ol lodash i guess. lodash >>> andi's utils
